server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -154,7 +152,7 @@ public ProgramCardParams getCard(
             .map(c -> c.getLocalizedName().getOrDefault(preferredLocale))
             .collect(ImmutableList.toImmutableList()));
 
-    String description = selectAndFormatDescription(program, preferredLocale);
+    String description = program.localizedShortDescription().getOrDefault(preferredLocale);
```

Comment:
suggestion (non-blocking): We might want to wrap the short description in a call to `removeMarkdown` like we do when it shows on the [admin side](https://github.com/civiform/civiform/blob/c64ec02e97817791c7812373e4c41281eea14b7f/server/app/views/components/ProgramCardFactory.java#L54). The bug you found when we were bug bashing program overview made me realize that there's nothing really preventing admin from entering markdown characters for short description, so we might want to guard against that by removing them before showing them to the applicant.

Or maybe we want to remove that call on the admin side to `removeMarkdown` so the unformatted markdown characters show up in both places. What do you think?

Reply:
Good call! Short description is used in many places (see #9719). If we want to always remove markdown, it would be better to do it a higher level place so it's consistent. I believe the place would be `ProgramDefinition.addLocalizedDescription`(https://sourcegraph.com/github.com/civiform/civiform@24d4ff0c701689e0a28a1ad507f016187c249a64/-/blob/server/app/services/program/ProgramDefinition.java?L876-879)

Otherwise, we could remove the `removeMarkdown` from the admin side. The idea is to be consistent either by always removing markdown or never.

We can discuss separately from this PR